model/src/main/scala/aqua/model/ValueModel.scala

```diff
@@ -18,6 +20,19 @@ sealed trait ValueModel {
 
 object ValueModel {
 
+  def errorCode(error: VarModel): Option[VarModel] =
+    error.intoField("error_code")
```

Comment:
you created `intoField` function just to do `val lastErrorCode = errorCode(lastError).get` with `.get`. Maybe do it in place?